recipes/scikit-garden/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "scikit-garden" %}
+{% set version = "0.1.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d6fe65205f39633f17c181f552a0fc56e2f05bb2096be63f38c160fe20718d9d" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - toolchain
+    - numpy 1.8.*  # [not (win and (py35 or py36))]
+    - numpy 1.9.*  # [win and py35]
+    - numpy 1.11.*  # [win and py36]
+
+  run:
+    - python
+    - numpy >=1.8  # [not (win and (py35 or py36))]
+    - numpy >=1.9  # [win and py35]
+    - numpy >=1.11  # [win and py36]
+    - scipy
+    - scikit-learn >=0.18
+
+test:
+  imports:
+    - skgarden
+    - skgarden.mondrian
+    - skgarden.quantile
+
+about:
+  home: https://scikit-garden.github.io/
+  license_file: {{ RECIPE_DIR }}/LICENSE.txt
```

Comment:
I just sent https://github.com/scikit-garden/scikit-garden/pull/47 so that packaging the license won't be necessary anymore; you could put a comment here referencing that so y'all remember to remove it in the future. :)